src/counterfactual_mean_based/gradient.jl

```diff
@@ -26,6 +26,94 @@ end
 
 plugin_estimate(ctf_aggregate) = mean(ctf_aggregate)
 
+function ccw_plugin_estimate(
```

Comment:
I think compute_gradient_and_estimate_from_caches! can be reused for these by providing a prevalence_weights keyword argument?